ALabel.tscn

```diff
@@ -15,5 +15,9 @@ margin_top = 94.4549
 margin_right = 252.983
 margin_bottom = 117.455
 custom_fonts/font = SubResource( 1 )
-text = "سلام"
+text = "ﻡﻼﺳ"
```

Comment:
This change doesn't look right. The letters are backward.

Reply:
It's as correct as it can be, dealing with software with no arabic support. It looks broken because, I assume, github displays arabic text correctly(RTL), whereas godot would display it backwards(LTR), thus displaying it as intended.

Reply:
It changed from سلام (salaam) to ﻡﻼﺳ (malaas), which looks ... wrong. :shrug: